protonvpn_cli/cli.py

```diff
@@ -326,20 +326,35 @@ def configure_cli():
 
         user_choice = user_choice.lower().strip()
         if user_choice == "1":
-            set_username_password(write=True)
+            try:
+                set_username_password()
+                time.sleep(1)
+            except KeyboardInterrupt:
+                print()
         elif user_choice == "2":
-            set_protonvpn_tier(write=True)
+            set_protonvpn_tier()
+            time.sleep(1)
+            print()
         elif user_choice == "3":
-            set_default_protocol(write=True)
+            set_default_protocol()
+            time.sleep(1)
+            print()
         elif user_choice == "4":
             set_dns_protection()
+            time.sleep(1)
+            print()
         elif user_choice == "5":
             set_killswitch()
+            time.sleep(1)
+            print()
         elif user_choice == "6":
             set_split_tunnel()
-        # Make sure this is always the last option
+            time.sleep(1)
+            print()
         elif user_choice == "7":
             purge_configuration()
+            break
```

Comment:
I've returned the "break" to the purge configuration option. If you'd purge the configuration, then be thrown back to the menu, all options would fail as there would be no profile initialized anymore.